web/client/epics/catalog.js

```diff
@@ -72,19 +72,33 @@ import { getResolutions, METERS_PER_UNIT } from "../utils/MapUtils";
 import { describeFeatureType } from '../api/WFS';
 import { extractGeometryType } from '../utils/WFSLayerUtils';
 import { createDefaultStyle } from '../utils/StyleUtils';
+import { removeDuplicateLines } from '../utils/StringUtils';
+
 const onErrorRecordSearch = (isNewService, errObj) => {
+    // Exception text is shown as is while the network errors are shown
+    // with generic error message in the notification
+    let [errorMsg] = castArray(errObj?.error);
+    if (errorMsg) {
+        // Remove any instance of duplicated line string from the exception text
+        errorMsg = removeDuplicateLines(errorMsg);
+    }
+    console.warn("Catalog error", errorMsg);
+
     if (isNewService) {
+        const message = errorMsg
+            ? truncate(errorMsg, { length: 200 })
```

Comment:
i think it would be better to try to recognize the error message and use a short generic message, and if not recognized it will use the previous one.

Reply:
I added it because we faced a similar case where notification occupied a whole app height when exception is lengthier.

@tdipisa @MV88 
Anyway, if the error message is of exception text, then we may show a generic message as **"Failed to save service, please check the attributes of the service"**. Your thoughts?

Reply:
@dsuren1 it would be good to have a precise message in this case otherwise it is unuseful. We could consider to leave it as it is but maybe increasing the length a bit more still to an acceptable value (eg. 400?). Let's see together @dsuren1. A solution could also be to improve the component to be expandable when needed but it should be a separated issue.